examples/linux-inotify.rs

```diff
@@ -37,17 +37,21 @@ fn main() -> std::io::Result<()> {
     future::block_on(async {
         // Watch events in the current directory.
         let mut inotify = Async::new(Inotify::init()?)?;
-        inotify
-            .get_mut()
-            .watches()
-            .add(".", WatchMask::ALL_EVENTS)?;
+
+        // SAFETY: We do not move the inner file descriptor out.
+        unsafe {
+            inotify
+                .get_mut()
+                .watches()
+                .add(".", WatchMask::ALL_EVENTS)?;
+        }
         println!("Watching for filesystem events in the current directory...");
```

Comment:
this should be using log (tracing?) instead of printing

Reply:
Yes, but that's not really relevant to this PR. Feel free to open a new one to fix this.